src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -29,69 +29,69 @@ class PactContractTest {
 
 
   @State("Server is healthy")
-  fun ensureServerIsHealthy() {}
+  fun `ensure server is healthy`() {}
 
   @State("Courses exist on the API")
-  fun ensureCoursesExist() {}
+  fun `ensure courses exist on the API` () {}
 
   @State("A course exists with ID 28e47d30-30bf-4dab-a8eb-9fda3f6400e8")
-  fun ensureCourseExists() {}
+  fun `ensure course exists with ID 28e47d30-30bf-4dab-a8eb-9fda3f6400e8`() {}
 
   @State("Offerings exist for a course with ID 28e47d30-30bf-4dab-a8eb-9fda3f6400e8")
-  fun ensureOfferingsExist() {}
+  fun `ensure offerings exist for a course with ID 28e47d30-30bf-4dab-a8eb-9fda3f6400e8`() {}
 
   @State("An offering exists with ID 20f3abc8-dd92-43ae-b88e-5797a0ad3f4b")
-  fun ensureOfferingExists() {}
+  fun `ensure an offering exists with ID 20f3abc8-dd92-43ae-b88e-5797a0ad3f4b`() {}
 
   @State("An offering with ID 20f3abc8-dd92-43ae-b88e-5797a0ad3f4b exists and has an associated course")
-  fun ensureOfferingExistsWithAssociatedCourse(){}
+  fun `ensure an offering with ID 20f3abc8-dd92-43ae-b88e-5797a0ad3f4b exists and has an associated course`(){}
 
   @State("participations exist for a person with prison number TPPIdBJ")
-  fun ensureParticipationsExistForPerson(){}
+  fun `ensure participations exist for a person with prison number TPPIdBJ`(){}
 
   @State("A request to destroy course participation 62d5c0be-edb9-47bb-89aa-7fc489e753d0")
-  fun requestToDestroyCourse(){}
+  fun `Request to destroy course participation with ID 62d5c0be-edb9-47bb-89aa-7fc489e753d0`(){}
 
   @State("A course participation exists with ID 62d5c0be-edb9-47bb-89aa-7fc489e753d0")
-  fun ensureCourseParticipationExists(){}
+  fun `Ensure course participation exists with ID 62d5c0be-edb9-47bb-89aa-7fc489e753d0`(){}
 
   @State("A course participation exists with ID 2480e7e3-5f9d-4e94-900c-148374ddbd72")
-  fun ensureCourseParticipationExistsForId_2480e7e3_5f9d_4e94_900c_148374ddbd72(){}
+  fun `ensure course participation exists with ID 2480e7e3-5f9d-4e94-900c-148374ddbd7`(){}
```

Comment:
Silly thing to mention, I know, but it's probably worth having our capitalisation be consistent here. Stick to either `ensure` or `Ensure` but don't use both! 😁

Reply:
I think you have reviewed an outdated file.